frontend/.eslintrc.yml

```diff
@@ -0,0 +1,19 @@
+env:
+  browser: true
+  es6: true
+plugins:
+  - prettier
+  - mozilla
+extends:
+  - standard
+  - prettier
+  - plugin:mozilla/recommended
+globals:
```

Comment:
Are these necessary?

Reply:
[Default settings from eslint](https://github.com/eslint/eslint/blob/4c0b70b869c16647f7af6de9d5c5479fc19f49db/lib/init/config-initializer.js#L272).
I trust those real JS dev to know what they do :smirk: